frontend/src/storybook/loader/loader.stories.ts

```diff
@@ -0,0 +1,77 @@
+import { type Meta, type StoryObj } from '@storybook/react';
+
+import { IconColor } from '~/bundles/common/components/icon/enums/icon-colors.enum.js';
+import { IconSize } from '~/bundles/common/components/icon/enums/icon-size.enum.js';
```

Comment:
Same here

Reply:
Oh thank you, that's the reason it was failing